test/sharness/t0051-object.sh

```diff
@@ -43,6 +43,26 @@ test_object_cmd() {
 		ipfs object get $HASH >actual_getOut
 	'
 
+	test_expect_success "'ipfs object get --encoding=protobuf' returns the correct content type" '
+		HASH="QmWkHFpYBZ9mpPRreRbMhhYWXfUhBAue3JkbbpFqwowSRb" &&
+    curl -sI "http://$API_ADDR/api/v0/object/get?arg=$HASH&encoding=protobuf" | grep -q "^Content-Type:\s*application/protobuf"
+  '
+
+	test_expect_success "'ipfs object get --encoding=json' returns the correct content type" '
+		HASH="QmWkHFpYBZ9mpPRreRbMhhYWXfUhBAue3JkbbpFqwowSRb" &&
```

Comment:
You don't need to re-set the `HASH` variable in each scope. shell variables like this are global(ish)